Dockerfile-python

```diff
@@ -1,5 +1,24 @@
-FROM python:3
+FROM python:3.5
```

Comment:
why did we have to fix the base image version?

Reply:
Because togeojsontiles is compatible with Python 3.3, 3.4, 3.5.